src/util/sort/min.js

```
@@ -12,7 +12,8 @@
 export default function min(arrMin) {
   function handleMin(arrMin) {
     return arrMin.filter(item => {
-      return Object.prototype.toString.call(item) === '[object Number]';
+      const newItem = item && typeof item === 'string' && Number(item) !== NaN ? Number(item) : item;
```

Biome (1.9.4)

[error] Use the Number.isNaN function to compare with NaN.

Unsafe fix: Use Number.isNaN() instead.


(lint/correctness/useIsNan)